x-pack/test/apm_api_integration/tests/alerts/alerting_api_helper.ts

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { SuperTest, Test } from 'supertest';
+import { ApmRuleType } from '@kbn/apm-plugin/common/rules/apm_rule_types';
+import { ApmRuleParamsType } from '@kbn/apm-plugin/common/rules/schema';
+
+export async function createIndexConnector({
+  supertest,
+  name,
+  indexName,
+}: {
+  supertest: SuperTest<Test>;
+  name: string;
+  indexName: string;
+}) {
+  const { body: createdAction } = await supertest
+    .post(`/api/actions/connector`)
+    .set('kbn-xsrf', 'foo')
+    .send({
+      name,
+      config: {
+        index: indexName,
+        refresh: true,
+      },
+      connector_type_id: '.index',
+    });
+  return createdAction.id as string;
+}
+
+export async function createAlertingRule<T extends ApmRuleType>({
```

Comment:
There are some places that could use the new helper ([service alerts](https://github.com/elastic/kibana/blob/main/x-pack/test/apm_api_integration/tests/services/service_alerts.spec.ts#L45), [service groups](https://github.com/elastic/kibana/blob/5d4c88057355062f92f01f570a0efc42e25b3483/x-pack/test/apm_api_integration/tests/service_groups/service_group_count/service_group_count.spec.ts#L29), [anomaly alert](https://github.com/elastic/kibana/blob/5d4c88057355062f92f01f570a0efc42e25b3483/x-pack/test/apm_api_integration/tests/alerts/anomaly_alert.spec.ts#L84))

Reply:
Align naming with type:
```suggestion
export async function createApmRule<T extends ApmRuleType>({
```